Bottom: 9bb0089ad8ea63881932854d11182a076d48bbe4
Top:    c75ee7a6bc218c86089d6587b0215be1a48d220c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 14:00:23 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index c3c2e4e..86bac04 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -281,7 +281,7 @@ gcode_choose_groups ()
     {
       layer = &PCB->Data->Layer[n];
 
-      if (layer->LineN || layer->TextN || layer->ArcN || layer->PolygonN)
+      if (layer->LineN || layer->TextN || layer->ArcN || layer->PourN)
 	{
 	  /* layer isn't empty */
